src/routes/vendor/vendor-show.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import fetch from 'isomorphic-fetch';
+
+import View from '../../components/vendor-show';
+
+class Vendor {
```

Comment:
if we're going to keep these classes inline can we move them to the bottom of the file with a brief comment about what they are and how they're used?